51114-0.txt

```diff
@@ -5153,7 +5153,7 @@ _Lorenzo_, I am determined to be his wife.
 _Cam._ How, sister! so sudden a turn? This inequality of temper indeed
 is not commendable.
 
-_Leo._ Your change, brother, is much more justly surprizing; you
+_Leo._ Your change, brother, is much more justly surprising; you

```

Comment:
"surprizing" used to be a common spelling:

https://books.google.com/ngrams/graph?content=surprizing%2Csurprising&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Csurprizing%3B%2Cc0%3B.t1%3B%2Csurprising%3B%2Cc0